src/components/trendingartists/TrendingArtists.jsx

```diff
@@ -88,7 +69,7 @@ function TrendingArtists() {
             </div>
             <div className='row'>
                 <div className='col-md-12'>
-                    <h1>Hi {userInfo.username}, here are some artists who have been gaining momentum recently and may be worth checking out:</h1>
+                    <h1>Hi {userInfo?.username}, here are some artists who have been gaining momentum recently and may be worth checking out:</h1>
```

Comment:
is it possible for username to be null or empty here? have you tested the behavior?

Reply:
The TrendingArtist.jsx screen is a protected route, meaning it can only be accessed by users who have signed up. Since a username is mandatory during the sign-up process, it is impossible for the username to be null under any circumstances. However, just for edge cases that I cant think of right now, I can add an if statement into the useEffect to check if the username exists and if not, it will display an empty string like this ""

Reply:
oh I actually don't think you need to do that, because adding the "?" after userInfo is an optional chaining operator. This means it will check if there is a username property before rendering and render nothing if not! I was wondering if it was necessary to have the "?" or not, and it sounds like not, but ofc it doesn't hurt to have

Reply:
Got it. Thanks!